src/app/containers/ATIAnalytics/params/media/index.test.jsx

```diff
@@ -35,24 +47,14 @@ describe('MediaAtiParams', () => {
     });
 
     it('should call atiPageViewParams', () => {
-      const mock = jest.fn().mockReturnValue('key=value&key2=value2');
-      atiPageViewParams.default = mock;
-
       render(Component(serviceContextStub, requestContextStub));
 
-      expect(mock).toHaveBeenCalledTimes(1);
-      expect(mock).toHaveBeenCalledWith({
```

Comment:
Wheres this stuff gone?

Reply:
https://github.com/bbc/simorgh/pull/3559/files#diff-eb5984997d72db8db652df216a4eb6ed

Since the `buildMediaATIUrl` is already tested, we should only test that it's called, and called with the right arguments.

Reply:
But nothings now testing the output, which this was doing?